viewer/src/main/java/nl/b3p/viewer/stripes/ApplicationActionBean.java

```diff
@@ -363,7 +377,7 @@ public Resolution view() throws JSONException, IOException {
 
         buildComponentSourceHTML(em);
 
-        appConfigJSON = application.toJSON(context.getRequest(),false, false,em);
+        appConfigJSON = application.toJSON(context.getRequest(),false, false,em).toString(4);
```

Comment:
why pretty-print this?

Reply:
Was already the case. We can do the regular toString() and don't have indentation, don't really care with developer tools being able to do that.